include/PROPOSAL/math/Cartesian3D.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "PROPOSAL/math/Vector3D.h"
+#include "PROPOSAL/json.hpp"
+
+namespace PROPOSAL{
+    class Spherical3D;
+    class Cartesian3D : public Vector3D {
+    public:
+        Cartesian3D() : Vector3D() {};
```

Comment:
If no further specified the default cstr is called. I think 
```
Cartesian3D() = default;
```
should be enough.